io/net/bonding.py

```diff
@@ -96,6 +96,10 @@ def setUp(self):
                                                 default=False)
         self.peer_wait_time = self.params.get("peer_wait_time", default=5)
         self.sleep_time = int(self.params.get("sleep_time", default=5))
+        self.ib = 'N'
+        if self.host_interface[0:2] == 'ib':
```

Comment:
Assumption is infiniband interfaces start only with ib ?
Are there any other cases ?

Reply:
Yeah Atleast hasn't changed in last 10 yrs.